yarn.lock

```diff
@@ -0,0 +1,2260 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This project is using `npm` – this file should not be committed.

Reply:
Very good point. Thank you